locator/token_metadata.cc

```diff
@@ -475,6 +482,7 @@ future<> token_metadata_impl::update_normal_tokens(const std::unordered_map<inet
             }
         }
     }
+    update_normal_token_owners();
```

Comment:
Not sure how much we care about it, but this function is not exception safe and may leave `_token_to_endpoint_map` partially updated and possibly `_nr_normal_token_owners` out of sync.

Reply:
yes, if token_metadata_impl::update_normal_tokens throws in the middle, we already have a broken _token_to_endpoint_map. It probably does not matter if _nr_normal_token_owners is in sync with _token_to_endpoint_map or not in this case.

Reply:
https://github.com/scylladb/scylla/issues/11127